save-demo/src/main/resources/diktat.cmd

```diff
@@ -0,0 +1,19 @@
+@echo off
```

Comment:
What's the difference between `ktlint.cmd`?

Reply:
Well if I got it right the text in these files is the same, but it depends on file name so it works fine